Tools/autotest/arducopter.py

```diff
@@ -1438,7 +1438,7 @@ def fly_simple(self, side=50):
         # switch to stabilize mode
         self.mavproxy.send('switch 6\n')
         self.wait_mode('STABILIZE')
-        self.set_rc(3, 1500)
+        self.set_rc(3, 1545)
```

Comment:
> This should probably be calling `hover()`. And hover() should probably be using 1545 rather than 1500 now :-)

no, the problem is the vehicle isn't level. When in STABILIZE and not level you need more throttle. The hover() function doesn't cover that. If we want to hover in STABILIZE we'd need to do our own alt control loop, or do what I've done here and work out what throttle is needed.